frontend/components/chat/ChatContainer.js

```diff
@@ -7,11 +7,12 @@ import Chat from './Chat';
 
 const mapStateToProps = ({
     profile: { username: userName },
-    chat: { log, status }
+    chat: { log, status, userNames }
 }) => ({
     userName,
     status,
-    log
+    log,
+    userNames
```

Comment:
I would name it somewhat else to distinguish from *userName* property. For instance, *onlineUsers*.